pynest/examples/arbor_cosim_example/nest_sender.py

```diff
@@ -14,6 +14,11 @@
 import nest
 
 
+STATUS_DICT = nest.ll_api.sli_func("statusdict")
+if (not  STATUS_DICT["have_arborbackend"]):
+    print ("Arbor_backend not available. Exit testscript!")
```

Comment:
In addition to @jougs suggestion this fixes the pep8 formatting.
```suggestion
if (not STATUS_DICT["have_recordingbackend_arbor"]):
    print("Recording backend Arbor not available. Exit testscript!")
```